src/views/CreateView.vue

```diff
@@ -299,7 +309,10 @@ export default {
     }
   },
   methods: {
-    ...mapActions(useExerciseStore, ['showSelectedExercises']),
+    showSelectedExercises(value) {
+      // eslint-disable-next-line max-len
+      return this.exercises.filter((exercise) => exercise.exercise.name.toLowerCase().includes(value))
```

Comment:
Use destruct in parameter.
```js
this.exercises,filter(({ exercise }) => exercise.name.toLowerCase().includes(value))
```

Reply:
I didn't actually need to destruct it after destructing the whole response in `mounted`.